tests/docker/ducker-ak

```diff
@@ -346,15 +362,23 @@ ducker_up() {
             -C|--custom-ducktape) set_once custom_ducktape "${2}" "the custom ducktape directory"; shift 2;;
             -f|--force) force=1; shift;;
             -n|--num-nodes) set_once num_nodes "${2}" "number of nodes"; shift 2;;
-            -j|--jdk) set_once jdk_version "${2}" "the OpenJDK base image"; shift 2;;
+            -j|--jdk) set_once jdk_version "${2}" "the default JDK version to use"; shift 2;;
+            -j11|--jdk11) set_once jdk_11_version "${2}" "the default JDK 11 version to use"; shift 2;;
+            -j11u|--jdk11-url) set_once jdk_11_url "${2}" "the default download JDK 11 url to use"; shift 2;;
+            -u|--url) set_once jdk_url "${2}" "the default download JDK url to use"; shift 2;;
+            -o|--os) set_once os_version "${2}" "the default OS version to use"; shift 2;;
             -e|--expose-ports) set_once expose_ports "${2}" "the ports to expose"; shift 2;;
             -m|--kafka_mode) set_once kafka_mode "${2}" "the mode in which kafka will run"; shift 2;;
             *) set_once image_name "${1}" "docker image name"; shift;;
         esac
     done
     [[ -n "${num_nodes}" ]] || num_nodes="${default_num_nodes}"
-    [[ -n "${jdk_version}" ]] || jdk_version="${default_jdk}"
+    [[ -n "${jdk_version}" ]] || jdk_version="${default_jdk}" || jdk_url="https://s3-us-west-2.amazonaws.com/kafka-packages/jdk-${default_jdk}.tar.gz"
```

Comment:
remove the jdk_version

Reply:
this `jdk_version` is not use in new Dockerfile, however it will use in naming image, should we retain this argument in order to change image name?
https://github.com/apache/kafka/blob/ff116df0152381425c98daf32ce78db3fa828063/tests/docker/ducker-ak#L358